src/scaict_uwu/libs/language/language_utils.py

```
@@ -0,0 +1,109 @@
+    def get_discord_code_to_bcp_47_mapping(cls) -> dict:
+        """
+        Get the mapping of Discord locale code to BCP 47 language tag.
+
+        Returns:
+            dict: The mapping of Discord locale code to BCP 47 language tag.
+        """
+
+        return {v: k for k, v in cls.get_discord_code_mapping()}
+
+    @classmethod
+    def get_discord_code(cls, code) -> str:
```

Ruff (0.8.0)

Missing type annotation for function argument `code`

(ANN001)